examples/emotion-css-prop/app/routes/index.tsx

```diff
@@ -0,0 +1,25 @@
+/** @jsx jsx */
```

Comment:
does esbuild respect this pragma?

Reply:
esbuild supports jsx annotations.
However, it seems that jsxImportSource support is not yet implemented without babel support.

ref: 
Support JSX annotation
https://github.com/evanw/esbuild/issues/138

Support /** @jsx pragma
https://github.com/evanw/esbuild/issues/389

Add support for jsxImportSource
https://github.com/evanw/esbuild/issues/1172

Reply:
Although it is limited, it seems to be possible to remove the pragma by adopting the patch in this issue.
- #1633